docs/source/gbd2017_models/risks/risk_exposure_template.rst

```diff
@@ -0,0 +1,53 @@
+.. _2017_risk_exposure_template:
+
+----------------------
+Risk Exposure Template
+----------------------
+
+.. todo::
+  Add instructions, following Nathaniel's cause model template instructions. Flesh out all headings.
+
+
+Risk Exposure Overview
+----------------------
+
+ - expert opinion
+
+Risk Exposure description
+-------------------------
+
+Risk Exposures Description in GBD

```

Comment:
I think there needs to be an added header underline before pushed to master, at least according to how it looks in the file on Git.

Reply:
Yes, the underline is too short - invalid `.rst`!

Also, do we need both the "Risk Exposure description" section and the "Risk Exposures Description in GBD" section?

Reply:
oops; thank you!